### PR TITLE
Register KeyEntryForCel for reflection inside Quarkus

### PR DESCRIPTION
**Pull request by dimas-b**
This is to fix native test failures from #4913

### CODE REVIEW

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java [hunk 1]
```diff
@@ -26,5 +26,11 @@
  * <p>In particular, this list is about Nessie model classes.
  */
 @RegisterForReflection(
-    targets = {NessieError.class, CELUtil.OperationForCel.class, CELUtil.ContentForCel.class})
+    targets = {
+      NessieError.class,
+      CELUtil.KeyedEntityForCel.class,
```
Comment (XN137) [inline servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:31]: just wondering:
`KeyedEntityForCel` is also new here, but mentioning int he commit msg is not important? :sweat_smile:

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java [hunk 2]
```diff
@@ -26,5 +26,11 @@
...
+      CELUtil.OperationForCel.class,
+      CELUtil.ContentForCel.class,
+      CELUtil.KeyEntryForCel.class
```
Comment (XN137) [inline servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:34]: seems like we could prevent this from happening in the future by automatically labeling PRs when one of these files changes i.e. `CELUtil.java`...
similar to:
https://github.com/apache/iceberg/blob/master/.github/labeler.yml

wdyt?

**Review by XN137 — APPROVED** [review]

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java (hunk 1 above)
Comment (dimas-b) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:31]: It is. I just mentioned the top think that was added in #4913. `KeyedEntityForCel` was a result of local refactoring, but it still has to be available for introspection.

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java (hunk 2 above)
Comment (dimas-b) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:34]: SGTM :+1: :)

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java (hunk 1 above)
Comment (dimas-b) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:31]: updated commit message for this.

**Review by snazy — APPROVED** [review]

File: servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java (hunk 2 above)
Comment (dimas-b) [reply servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java:34]: #4931

**Pull request merged**